docs/en/ingest-management/elastic-agent/elastic-agent-unprivileged-mode.asciidoc

```diff
@@ -0,0 +1,125 @@
+[[elastic-agent-unprivileged]]
+= Run {agent} without administrative privileges
+
+Beginning with {stack} version 8.15, {agent} is no longer required to be run by a user with superuser privileges. You can now run agents in an `unprivileged` mode that does not require `root` access on Linux or macOS or `admin` access on Windows. Being able to run agents without full administrative privileges is often a requirement in organizations where this kind of access is often very limited.
+
+In general, agents running without full administrative privileges will perform and behave exactly as those run by a superuser. There are certain integrations and datastreams that are not available, however. If an integration requires root access, this is <<unprivileged-integrations,indicated on the integration main page>>.
+
+// Add mention of the System integration data streams.
+
+You can also <<unprivileged-change-mode,change the privilege mode>> of an {agent} after it has been installed.
+
+Refer to <<unprivileged-command-behaviors>> and <<unprivileged-running>> for the requirements and steps associated with running an agent in `unprivileged` model.
+
+* <<unprivileged-running>>
+* <<unprivileged-command-behaviors>>
+* <<unprivileged-integrations>>
+* <<unprivileged-view-mode>>
+* <<unprivileged-change-mode>>
+
+[discrete]
+[[unprivileged-running]]
+== Run {agent} in `unprivileged` mode
+
+TBD
```

Comment:
@blakerouse Could you provide here the steps a user would need to take to install/run Agent in unprivileged mode please?  Thanks!

Reply:
All the commands remain the same except for `install` which has the new `--unprivileged` flag. Lets say in Fleet it says run `elastic-agent install --url <URL> --enrollment-token <TOKEN>` then you just need to add an additional `--unprivileged` flag to get it to install in unprivileged mode.

Reply:
Here's what I added:

![Screenshot 2024-06-20 at 1 52 20 PM](https://github.com/elastic/ingest-docs/assets/41695641/e8197349-28a2-4630-916c-d294151ff4a9)